docs/serving/autoscaling/README.md

```diff
@@ -1,22 +1,17 @@
 # Autoscaling
 
-One of the main features of Knative is automatic scaling of replicas for an application to closely match incoming demand, including scaling applications to zero if no traffic is being received.
-Knative Serving enables this by default, using the Knative Pod Autoscaler (KPA).
-The Autoscaler component watches traffic flow to the application, and scales replicas up or down based on configured metrics.
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. This is provided by default, by using the Knative Pod Autoscaler (KPA).
 
-Knative services default to using autoscaling settings that are suitable for the majority of use cases. However, some workloads may require a custom, more finely-tuned configuration.
-This guide provides information about configuration options that you can modify to fit the requirements of your workload.
+For example, if an application is receiving no traffic, and scale to zero is enabled, Knative Serving scales the application down to zero pods. If scaling to zero is disabled, the application is scaled down to the minimum number of pods specified for applications on the cluster. Pods can also be scaled up to meet demand if traffic to the application increases.
```

Comment:
nits: 

> For example, if an application is receiving no traffic, and scale to zero is enabled, 

For example, if an application is receiving no traffic and scale to zero is enabled, 
(remove comma)

> Pods can also be scaled up to meet demand if traffic to the application increases.

Pods will also be...
(to me, "can" implies it's something the user has to configure)

Reply:
"Pods are" maybe? 🙂

Reply:
works for me!